ykman-gui/py/yubikey.py

```diff
@@ -14,17 +14,18 @@
 from fido2.ctap import CtapError
 
 from ykman.descriptor import get_descriptors
+from ykman.driver_ccid import (APDUError, SW)
 from ykman.driver_otp import YkpersError
 from ykman.device import device_config
 from ykman.otp import OtpController
 from ykman.fido import Fido2Controller
-from ykman.piv import PivController
+from ykman.piv import (PivController)
+from ykman.piv import (AuthenticationBlocked, WrongPin, WrongPuk)
```

Comment:
Drop parenthesis if not multi line import?

Reply:
Sure!